app/views/catalog/_home_text.html.erb

```diff
@@ -34,10 +34,10 @@
   </div>
   <div class="covid19-text">
     <%= link_to "https://library.princeton.edu/phased-planning-updates" do %>
-      <h3>New Services: Pick-up and Digitization</h3>
+      <h3>New Services: Study-Browse Service, Pick-up, and Digitization</h3>
     <% end %>
     <p>
-    Firestone and most branch libraries are now offering pick-up service and performing in-house digitization to fulfill requests. Catalog records for many items will now include a button you can use to request pick-up of circulating items, or digitization of items.
+    Continuing for the Spring semester, access to Library spaces is available via our Study-Browse Service. Firestone and most branch libraries are also continuing to offer pick-up service and performing in-house digitization to fulfill requests. Catalog records for many items will now include a button you can use to request pick-up of circulating items, or digitization of items.
```

Comment:
Can you make "Study-Browse Service" a link to https://library.princeton.edu/services/study-browse ?